src/wizard2.cc

```diff
@@ -1694,11 +1693,6 @@ void do_cmd_debug()
 		gain_random_corruption();
 		break;
 
-		/* Create a trap */
-	case 'R':
```

Comment:
Speaking of which, maybe I should also fix up the manual re: skill trees for affected classes.

(Wouldn't it be nice if the documentation would built into the code base?)